From aef048fedb7da432e023d56d74c54be07e91bed0 Mon Sep 17 00:00:00 2001
Message-Id: <patch-v4-03.34-aef048fedb7-20211117T101025Z-avarab@gmail.com>
In-Reply-To: <cover-v4-00.34-00000000000-20211117T101025Z-avarab@gmail.com>
References: <cover-v3-00.23-00000000000-20211116T114334Z-avarab@gmail.com>
	<cover-v4-00.34-00000000000-20211117T101025Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Fri, 26 Mar 2021 18:42:08 +0100
Subject: [PATCH v4 03/34] branch docs: include the branch.* section from
 config
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Include the branch.* section from git-config(1) in the git-branch
docs, similarly to what I did in b6a8d09f6d8 (gc docs: include the
"gc.*" section from "config" in "gc", 2019-04-07).

Unlike the just-introduced template for saying the entire section is
included, in this case we have some discussion unique to the
documentation earlier in it. Let's add a standard text for that into
includes/* for later use in other similar scenarios.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 Documentation/git-branch.txt                       | 4 ++++
 Documentation/includes/cmd-config-section-rest.txt | 3 +++
 2 files changed, 7 insertions(+)
 create mode 100644 Documentation/includes/cmd-config-section-rest.txt

diff --git a/Documentation/git-branch.txt b/Documentation/git-branch.txt
index 8af42eff895..b1f1e8ce41c 100644
--- a/Documentation/git-branch.txt
+++ b/Documentation/git-branch.txt
@@ -305,6 +305,10 @@ CONFIGURATION
 `--list` is used or implied. The default is to use a pager.
 See linkgit:git-config[1].
 
+include::templates/configuration-include-rest-heading.txt[]
+
+include::config/branch.txt[]
+
 EXAMPLES
 --------
 
diff --git a/Documentation/includes/cmd-config-section-rest.txt b/Documentation/includes/cmd-config-section-rest.txt
new file mode 100644
index 00000000000..fe4721dac76
--- /dev/null
+++ b/Documentation/includes/cmd-config-section-rest.txt
@@ -0,0 +1,3 @@
+The discussion above this line is unique to this documentation, but
+everything below is included from section(s) in linkgit:git-config[1]
+aplicable to this command:
-- 
2.34.0.796.g126c2ced91c

